test-setup.sh

```
@@ -3,11 +3,11 @@
 set -ex
 
 # create test namespace
-oc get namespace ${CYPRESS_TEST_NS} || oc new-project ${CYPRESS_TEST_NS}
+oc get namespace ${TEST_NS} || oc new-project ${TEST_NS}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)